examples/grpc-client/main.go

```diff
@@ -1,3 +1,4 @@
+//go:build ignore
```

Comment:
I think these came up on running `make tidy`. Do we need this ?

Reply:
I don't think so. Can we revert all files that have nothing to do with the change? Not sure what the `go.sum` change is as well.

Reply:
Okay so these are generated `make fmt`. Removing it for now.